earth_enterprise/src/third_party/apache2/apache-ge-httpd-ssl.conf.in.patch

```diff
@@ -18,7 +18,7 @@
 -SSLProtocol all -SSLv3
 +#SSLProtocol all -SSLv3
 +# Limit SSL protocols.
-+SSLProtocol -ALL +TLSv1.2 +SSLv3 +TLSv1
++SSLProtocol -ALL +TLSv1.2 +TLSv1
```

Comment:
Maybe this is beyond the scope of the ticket, but should we also consider disabling TLSv1? Even the comment above this line says that it shouldn't be used past 2016. We kept it longer because older versions of EC didn't support TLSv1.2, but that should have been taken care of for a long time now, and if a customer still has a reason to use it, that should be a deliberate choice to use an unsecure protocol rather than the default.

Reply:
That's a good point. We only removed SSLv3 because that's the one we're forced to remove to upgrade to Open SSL 1.1.1. I could see taking out TLSv1 as well if we're certain it's not going to cause problems.

Reply:
Hmmm, I'm not sure I'd say I'm certain. I can look into whether we have any statistics we can refer to.